decidim-conferences/spec/commands/create_conference_spec.rb

```diff
@@ -146,19 +146,6 @@ module Decidim::Conferences
         linked_consultations = conference.linked_participatory_space_resources("Consultations", "included_consultations")
         expect(linked_consultations).to match_array(questions.collect(&:consultation).uniq)
       end
-
-      it "notifies the change" do
```

Comment:
Shouldn't this spec be moved to the `publish_conference_spec`?

Reply:
Ups sorry! I forgot to commit the change

Reply:
@oriolgual requested change applied!